src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -144,7 +152,25 @@ function isPaymentMethodActive(actionPaymentMethodType, activePaymentMethodID, p
     return paymentMethod.accountType === actionPaymentMethodType && paymentMethod.methodID === activePaymentMethodID;
 }
 function PaymentMethodList(props) {
-    const {actionPaymentMethodType, activePaymentMethodID, bankAccountList, fundList, filterType, network, onPress, shouldShowSelectedState, selectedMethodID, translate} = props;
+    const {
+        actionPaymentMethodType,
```

Comment:
NAB, maybe better to deconstruct this in the args if we are not going to reference the props directly. I think I have seen that more than what we are doing here.